packages/server/src/utils/types.ts

```diff
@@ -38,12 +38,25 @@ export interface DatabaseConfig {
 	client: DatabaseConfigClient;
 	// For Postgres, this is the actual database name. For SQLite, this is the
 	// path to the SQLite file.
-	name: string;
+	name?: string;
+	connectionString?: string;
 	host?: string;
 	port?: number;
 	user?: string;
 	password?: string;
 	asyncStackTraces?: boolean;
+	sslCaFilePath?: string;
+	sslCertFilePath?: string;
+	sslCertKeyFilePath?: string;
+	rejectUnauthorized?: boolean;
+}
+
+export interface SslDatabaseConfig
```

Comment:
For consistency, it should be DbConfigConnectionSsl (parent interface name first).